client/build.gradle

```diff
@@ -1,3 +1,57 @@
+plugins {
+    id "idea"
+}
+
+apply plugin: "com.google.protobuf"
+
+protobuf {
+    protoc {
+        artifact = 'com.google.protobuf:protoc'
+    }
+
+    generatedFilesBaseDir = "$projectDir/generated"
+
+    plugins {
+        grpc {
+            artifact = 'io.grpc:protoc-gen-grpc-java'
```

Comment:
shouldn't be that different deps? Like client-grpc, client-rsocket?

Reply:
can be, in future. Not going to change in this PR, for backward compatibility of the client